### PR TITLE
improve scaladoc for DoNotInherit

### PR DESCRIPTION
**Pull request by dongxuwang**

### CODE REVIEW

**Review by pjfanning — DISMISSED** [review]
lgtm - thanks

File: actor/src/main/java/org/apache/pekko/annotation/DoNotInherit.java [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
- * Marks APIs that are designed under an closed-world assumption for and are NOT meant to be
+ * Marks APIs that are designed under a closed-world assumption for and are NOT meant to be
  * extended by user-code. It is fine to extend these classes within Apache Pekko itself, however.
  *
  * <p>This is most useful for binary compatibility purposes when a set of classes and interfaces
  * assume a "closed world" between them, and gain the ability to add methods to the interfaces
  * without breaking binary compatibility for users of this code. Specifically this assumption may be
  * understood intuitively: as all classes that implement this interface are in this compilation unit
- * / artifact, it is impossible to obtain a "old" class with a "new" interface, as they are part of
+ * / artifact, it is impossible to obtain an "old" class with a "new" interface, as they are part of
```
Comment (He-Pin) [inline actor/src/main/java/org/apache/pekko/annotation/DoNotInherit.java:26]: need a `javafmtAll`.

**Review by pjfanning — APPROVED** [review]
lgtm

**Comment by pjfanning** [comment]
merged - thanks

**Pull request merged**